txt2tags.py

```diff
@@ -1898,9 +1901,12 @@ def Readfile(file_path):
 
 def Savefile(file_path, lines):
     try:
-        with open(file_path, "w") as f:
+        with io.open(file_path, "w", encoding="utf-8") as f:
             for line in lines:
-                f.write(line + "\n")
+                if is_python2:
```

Comment:
But since this function write the file line by line (why, btw?), for each line it will try and fail, raise the exception, and then try again with the proper encoding. Isn't that ineficient?

What about adding the line breaks first, and then saving the file in a single step? I did that in the most recent change.